reference/release-notes/1.28.1.md

```diff
@@ -0,0 +1,47 @@
+---
+orphan: true
+---
+# 1.28.1
+
+These release notes cover new features and changes in Anbox Cloud 1.28.1.
+
+Anbox Cloud 1.28.1 is a patch release. To understand minor and patch releases, see [Release notes](https://documentation.ubuntu.com/anbox-cloud/en/latest/reference/release-notes/release-notes).
+
+Please see [Component versions](https://documentation.ubuntu.com/anbox-cloud/en/latest/reference/component-versions/) for a list of updated components.
+
+## Requirements
+
+See the [Requirements](https://documentation.ubuntu.com/anbox-cloud/en/latest/reference/requirements/) for details on general and deployment specific requirements to run Anbox Cloud.
+
+## New features & improvements
+
+* The LXD charm in Anbox Cloud can be configured to use [Ceph storage](https://ceph.io/en/) in addition to the default local storage options. This helps deployments that require shared storage, improved scalability and better resource utilization.
+* Android security updates for December 2025 (see [Android Security Bulletin - December 2025](TBD)). 
+* The Android WebView has been updated to [142.0.7444.48](https://chromereleases.googleblog.com/2025/10/chrome-beta-for-android-update_22.html).
+
+## Known issues
+
+See our [open bugs in Launchpad](https://bugs.launchpad.net/anbox-cloud/+bugs?field.searchtext=&orderby=-importance&field.status%3Alist=NEW&field.status%3Alist=CONFIRMED&field.status%3Alist=TRIAGED&field.status%3Alist=INPROGRESS&field.status%3Alist=INCOMPLETE_WITH_RESPONSE&field.status%3Alist=INCOMPLETE_WITHOUT_RESPONSE&assignee_option=any&field.assignee=&field.bug_reporter=&field.bug_commenter=&field.subscriber=&field.structural_subscriber=&field.tag=&field.tags_combinator=ANY&field.has_cve.used=&field.omit_dupes.used=&field.omit_dupes=on&field.affects_me.used=&field.has_patch.used=&field.has_branches.used=&field.has_branches=on&field.has_no_branches.used=&field.has_no_branches=on&field.has_blueprints.used=&field.has_blueprints=on&field.has_no_blueprints.used=&field.has_no_blueprints=on&search=Search).
+
+## CVEs
+
+The fixes for the following CVEs are included with the 1.28.1 release:
+
+
+| CVE | Affected components |
+|-----|-------------------|
+| [CVE-2025-58181](https://ubuntu.com/security/CVE-2025-58181) | Anbox |
+| [CVE-2025-47914](https://ubuntu.com/security/CVE-2025-47914) | Anbox |
```

Comment:
```suggestion
| [CVE-2025-47914](https://ubuntu.com/security/CVE-2025-47914) | Anbox |
| [CVE-2025-64751](https://github.com/advisories/GHSA-2c64-vmv2-hgfc) | AMS |
```

I didn't put an Ubuntu security link as it's very light on info, see https://ubuntu.com/security/CVE-2025-64751